tests/unit/types/score/test_score.py

```diff
@@ -0,0 +1,27 @@
+from jina.types.score import NamedScore
+
+
+def test_named_score():
+    score = NamedScore({'op_name': 'operation',
+                        'value': 10.0,
+                        'ref_id': '10',
+                        'description': 'score description'})
+
+    assert score.op_name == 'operation'
+    assert score.value == 10.0
+    assert score.ref_id == '10'
+    assert score.description == 'score description'
+
+
+def test_named_score_setters():
+
+    score = NamedScore()
+    score.op_name = 'operation'
+    score.value = 10.0
+    score.ref_id = '10'
+    score.description = 'score description'
+
+    assert score.op_name == 'operation'
+    assert score.value == 10.0
+    assert score.ref_id == '10'
+    assert score.description == 'score description'
```

Comment:
Another test to instantiate from `jina_pb2.NamedScoreProto`?

Reply:
perf!

Reply:
Done!